src/app/js/copyright-year.js

```diff
@@ -0,0 +1,3 @@
+const copyrightYear = document.getElementById('copyright-year');
+
+copyrightYear.innerText = new Date().getFullYear().toString();
```

Comment:
The copyright text is usualy year-year so maybe do the first value to be 2021-{calulated value based on current year}

Reply:
@PlamenMichev please check now